chunjun-core/src/main/java/com/dtstack/chunjun/util/JobUtil.java

```diff
@@ -48,13 +48,29 @@ public static String JsonValueReplace(String json, Map<String, String> parameter
         return json;
     }
 
-    /** 将命令行中的修改命令转化为HashMap保存 */
+    /**
+     * @param command
+     * @return HashMap
+     * @description: convert params after '-p' in shell to hashmap
+     */
     public static HashMap<String, String> CommandTransform(String command) {
         HashMap<String, String> parameter = new HashMap<>();
         String[] split = StringUtils.split(command, ConstantValue.COMMA_SYMBOL);
-        for (String item : split) {
-            String[] temp = item.split(ConstantValue.EQUAL_SYMBOL);
-            parameter.put(temp[0].trim(), temp[1].trim());
+        for (int i = 0; i < split.length; i++) {
+            String[] temp = split[i].split(ConstantValue.EQUAL_SYMBOL);
```

Comment:
I don't believe that a parameter named 'temp' is readable.

Reply:
agree. I will make a change